lib/slides_page.dart

```diff
@@ -25,25 +25,32 @@ import 'slide.dart';
 /// An 'installation status' is (planned to be) shown at the bottom of the page.
 class SlidesPage extends StatelessWidget {
   final List<Slide> slides;
+  final Widget bottom;
 
-  const SlidesPage(this.slides, {Key? key}) : super(key: key);
+  const SlidesPage(this.slides, {required this.bottom, Key? key})
+      : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
```

Comment:
i think this is already the default

Reply:
Indeed. In between experiments that was a leftover. Removed.